internal/engine/eval_status.go

```diff
@@ -98,6 +98,16 @@ func (e *Executor) createEvalStatusParams(
 	return params, nil
 }
 
+// createOrUpdateEvalStatus takes care of recording the rule evaluation results.
+// This function inserts into the database:
+//
+//   - The rule evaluation parameters (profile, repo, artifact, entity, etc).
+//   - The rule evaluation status and details.
+//   - The remediation status and details.
+//   - The alert status and details.
+//
+// If the error in the evaluation status resolves to an errors.ErrEvaluationSkipSilently,
+// no details are stored or logged.
```

Comment:
I just noticed I pulled an unchecked check from another branch I was working on.. I'll leave it in the PR, it is only expanding the function documentation elsewhere.